src/transformation/builtins/math.ts

```diff
@@ -74,7 +68,13 @@ export function transformMathCall(context: TransformationContext, node: Property
         case "acos":
         case "asin":
         case "atan":
-        case "atan2":
+        case "atan2": {
+            if (expressionName === "atan2" && context.options.luaTarget === LuaTarget.Lua53) {
```

Comment:
`expressionName === "atan2"` is already checked by the switch, it's not required here

Reply:
Oh I see why it's here now, it's still trange to me, you can move atan2 up to be the first case or something